### PR TITLE
Show minimal ETH amount in TopUpWithFiat

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: universal-login-react/src/ui/TopUp/Fiat/FiatFooter.tsx [hunk 1]
```diff
@@ -29,6 +30,9 @@ export const FiatFooter = ({paymentMethod}: FiatFooterProps) => {
           <div className="info-block info-row">
             <p className="info-text info-text-hint">Minimum amount is 1£</p>
           </div>
+          <div className="info-block info-row">
+            <p className="info-text info-text-hint">Minimum amount required to deploy contract is {minimalAmount} ETH</p>
+          </div>
```
Comment (xitronix) [inline universal-login-react/src/ui/TopUp/Fiat/FiatFooter.tsx]: We should do not show this info if it is not deployment and probably use same info as in crypto (DeploymentWithCryptoInfo)

Comment (cehali) [reply universal-login-react/src/ui/TopUp/Fiat/FiatFooter.tsx]: 👍

File: universal-login-react/src/ui/TopUp/TopUpWithCrypto.tsx [hunk 2]
```diff
@@ -55,7 +50,7 @@ export const TopUpWithCrypto = ({contractAddress, isDeployment, minimalAmount}:
             />
           </div>
           {isDeployment
-            ? <DeploymentWithCryptoInfo minimalAmount = {minimalAmount}/>
+            ? <MinimalAmountInfo minimalAmount={minimalAmount} paymentMethod='crypto' />
             : <TopUpCryptoInfo/>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/TopUp/TopUpWithCrypto.tsx]: This one should be also shown in the deployment?

Comment (cehali) [reply universal-login-react/src/ui/TopUp/TopUpWithCrypto.tsx]: It hasn't been till today so I don't think so

**Pull request closed without merging**